libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -25,8 +25,39 @@
         </service>
 
         <!-- Login activity -->
+        <!--
+            Launch mode of "singleInstance" ensures that the activity isn't restarted
+            by a callback from Chrome custom tab when auth flow is complete. This is
+            required for the Chrome custom tab auth flow to work correctly.
+        -->
         <activity android:name="com.salesforce.androidsdk.ui.LoginActivity"
-            android:theme="@style/SalesforceSDK.ActionBarTheme" />
+            android:theme="@style/SalesforceSDK.ActionBarTheme"
+            android:launchMode="singleInstance">
```

Comment:
We need `launchMode` to be `singleInstance` to ensure it gets dismissed after login.